### ISSUE TITLE
Seeing a warning: naether-0.15.1-java/lib/naether/configuration.rb:25: warning: key (SymbolNode:version 28) is duplicated and overwritten on line 25

### ISSUE THREAD

**Issue by xxx** [mguymon/naether#26]
it's the `:version` key, if I'm looking in the right place, though I don't see how it's being overwritten. I see this warning every time I run a script that loads jars via LockJar though.

**Comment by stewi2** [comment]
the key just appears in the hash twice, and the warning just says that the second value overwrites the first one:

```
      @data = {
        :version => version,
        :gem_dir =>     gem_dir,
        :naether_jar => File.join( gem_dir, "core-#{version}.jar"),
        :platform =>    ($platform || RUBY_PLATFORM[/java/] || 'ruby'),
        :version =>     version,
        :dependencies_yml => File.expand_path("#{File.dirname( __FILE__ )}/../../jar_dependencies.yml")
      }
```

So the fix here is to just remove one if these two.

**Comment by stewi2** [comment]
this seems to be a new warning in JRuby 9.1.2.x / Ruby 2.2 and up

**Comment by mgold** [comment]
Any chance this can get merged and released? It prints an annoying warning message every time you start the JVM.

**Comment by mguymon** [comment]
Released with version 0.15.4
